centreon/www/front_src/src/Resources/Filter/FilterStorage.cypress.spec.tsx

```diff
@@ -247,7 +416,7 @@ describe('Filter storage', () => {
     cy.findByLabelText(labelSearchOptions).click();
 
     cy.findByTestId(labelHost).click();
-    cy.waitForRequest('@getResourcesByHostType');
+
```

Comment:
@coderabbitai emm but with the hardcoded variables , if i update the testId i should update all the hardcoded variables , otherwise with labelHost i will do the change in one place !